packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/document/dto/UploadDocumentInput.java

```diff
@@ -24,5 +24,8 @@
  *
  * @author hmarchadour
  */
-public record UploadDocumentInput(@NotNull UUID id, @NotNull String editingContextId, @NotNull UploadFile file, boolean readOnly) implements IInput {
+public record UploadDocumentInput(@NotNull UUID id, @NotNull String editingContextId, @NotNull UploadFile file, boolean allowProxies, boolean readOnly) implements IInput {
+    public UploadDocumentInput(@NotNull UUID id, @NotNull String editingContextId, @NotNull UploadFile file, boolean readOnly) {
+        this(id, editingContextId, file, false, readOnly);
+    }
 }
```

Comment:
This is mainly used to communicate between the frontend and backend. It would be very odd to use it to carry strictly internal data hidden from the GraphQL schema. I'll have to retrieve the code and look at it in more details to understand the flow of the code.